cmd/katapult/networks_test.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/krystal/go-katapult"
+	"github.com/krystal/go-katapult/core"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var idNetworks = []*core.Network{
+	{
+		ID:        "pognet",
+		Name:      "Pognet 1",
+		Permalink: "pog-1",
+		DataCenter: &core.DataCenter{
+			ID:        "POG1",
+			Name:      "Pogland 1",
+			Permalink: "pog1",
+			Country: &core.Country{
+				ID:   "pog",
+				Name: "Pogland",
+			},
+		},
+	},
+	{
+		ID:        "pognet2",
+		Name:      "Pognet 2",
+		Permalink: "pog-2",
+		DataCenter: &core.DataCenter{
+			ID:        "POG1",
+			Name:      "Pogland 1",
+			Permalink: "pog1",
+			Country: &core.Country{
+				ID:   "pog",
+				Name: "Pogland",
+			},
+		},
+	},
+}
+
+var subdomainNetworks = []*core.Network{
+	{
+		ID:        "pognet3",
+		Name:      "Pognet 3",
+		Permalink: "pog-3",
+		DataCenter: &core.DataCenter{
+			ID:        "POG1",
+			Name:      "Pogland 1",
+			Permalink: "pog1",
+			Country: &core.Country{
+				ID:   "pog",
+				Name: "Pogland",
+			},
+		},
+	},
+	{
+		ID:        "pognet4",
+		Name:      "Pognet 4",
+		Permalink: "pog-4",
+		DataCenter: &core.DataCenter{
+			ID:        "POG1",
+			Name:      "Pogland 1",
+			Permalink: "pog1",
+			Country: &core.Country{
+				ID:   "pog",
+				Name: "Pogland",
+			},
+		},
+	},
+}
+
+type mockNetworkList struct{}
+
+func (mockNetworkList) List(
+	_ context.Context, org core.OrganizationRef,
+) ([]*core.Network, []*core.VirtualNetwork, *katapult.Response, error) {
+	switch {
+	case org.SubDomain == "pog-subdomain":
+		return subdomainNetworks, nil, nil, nil
+	case org.ID == "pog-id":
+		return idNetworks, nil, nil, nil
+	default:
+		return nil, nil, nil, katapult.ErrNotFound
+	}
+}
+
+func TestNetworkList(t *testing.T) {
```

Comment:
Nitpick: Keep test function names consistent.

```suggestion
func TestNetworks_List(t *testing.T) {
```